### DISCUSSION TITLE
Adding an option to clear date value of input box

### DISCUSSION THREAD

**Discussion by AtousaDarabi** [formkit/formkit#890] [Ideas]
Need to delete date in datepicker in a way  which is not backspace in three steps (delete day, delete month, delete year)
For example adding an X  beside calendar icon in field and by clicking it remove the date or have a "Clear" button in datepicker's dialogue box or any other way...

**Comment by js-modernary** [comment]
FYI - this has been implemented - [https://formkit.com/inputs/datepicker#clearable](https://formkit.com/inputs/datepicker#clearable)
